ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -0,0 +1,142 @@
+    SELECT 1 FROM security_profile_info WHERE minimum_password_length >= 14;
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS1.1.4
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Password must meet complexity requirements' is set to 'Enabled' 
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting checks all new passwords to ensure that they meet basic requirements for
+    strong passwords. Passwords that contain only alphanumeric characters are extremely easy to
+    discover with several publicly available tools.
+  resolution: |
+    Manual method:
+    Ask your system administrator to establish the recommended configuration via GP, set the
+    following UI path to 'Enabled':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Account Policies\Password Policy\Password must meet complexity requirements'
+  query: |
+    SELECT 1 FROM security_profile_info WHERE password_complexity = 1;
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS1.1.5
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Password must meet complexity requirements' is set to 'Enabled' 
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting determines whether the minimum password length setting can be increased beyond the legacy limit of 14 characters.
+  resolution: |
+    Manual method:
+    Ask your system administrator to establish the recommended configuration via GP, set the
+    following UI path to 'Enabled':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Account Policies\Password Policy\Relax minimum password length limits'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\SAM\\RelaxMinimumPasswordLengthLimits' AND data != 0);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS1.1.6
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Store passwords using reversible encryption' is set to 'Disabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting determines whether the operating system stores passwords in a way that uses
+    reversible encryption, which provides support for application protocols that require knowledge
+    of the user's password for authentication purposes. Passwords that are stored with reversible
+    encryption are essentially the same as plaintext versions of the passwords.
+  resolution: |
+    Manual method:
+    Ask your system administrator to establish the recommended configuration via GP, set the
+    following UI path to 'Disabled':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Account Policies\Password Policy\Store passwords using reversible encryption'
+  query: |
+    SELECT 1 FROM security_profile_info WHERE clear_text_password = 1;
```

Comment:
Am confused by this policy check:

We want the feature `'Store passwords using reversible encryption'` to be disabled and prefer clear text passwords? (Guessing the encryption is weak/broken, causes issues, so why bother, just use clear text storage?)

Reply:
you are right. I got confused while creating this. The check should be set to 0 instead of 1 as the documentation states that the policy should be in `Disabled` state